test/test_tables.py

```diff
@@ -188,6 +213,67 @@ def test_sort_by():
     assert sorted2.x[2].as_py() == 1
 
 
+def test_sort_by_nested():
+    pair = Pair.from_kwargs(x=[1, 2, 3], y=[5, 1, 2])
+    wrapper = Wrapper.from_kwargs(
+        id=["1", "2", "3"],
+        pair=pair,
+    )
+
+    sorted1 = wrapper.sort_by("pair.y")
+    assert sorted1.id[0].as_py() == "2"
+    assert sorted1.id[1].as_py() == "3"
+    assert sorted1.id[2].as_py() == "1"
+    assert sorted1.pair.x[0].as_py() == 2
+    assert sorted1.pair.x[1].as_py() == 3
+    assert sorted1.pair.x[2].as_py() == 1
+    assert sorted1.pair.y[0].as_py() == 1
+    assert sorted1.pair.y[1].as_py() == 2
+    assert sorted1.pair.y[2].as_py() == 5
+
+    sorted2 = wrapper.sort_by([("pair.x", "descending")])
+    assert sorted2.id[0].as_py() == "3"
+    assert sorted2.id[1].as_py() == "2"
+    assert sorted2.id[2].as_py() == "1"
+    assert sorted2.pair.x[0].as_py() == 3
+    assert sorted2.pair.x[1].as_py() == 2
+    assert sorted2.pair.x[2].as_py() == 1
+    assert sorted2.pair.y[0].as_py() == 2
+    assert sorted2.pair.y[1].as_py() == 1
+    assert sorted2.pair.y[2].as_py() == 5
+
+
+def test_sort_by_nested_doubly():
+    class DoublyNested(qv.Table):
+        inner = Wrapper.as_column()
+
+    dn = DoublyNested.from_kwargs(
+        inner=Wrapper.from_kwargs(id=["1", "2", "3"], pair=Pair.from_kwargs(x=[1, 2, 3], y=[5, 1, 2]))
+    )
+
+    sorted1 = dn.sort_by("inner.pair.y")
+    assert sorted1.inner.id[0].as_py() == "2"
+    assert sorted1.inner.id[1].as_py() == "3"
+    assert sorted1.inner.id[2].as_py() == "1"
+    assert sorted1.inner.pair.x[0].as_py() == 2
+    assert sorted1.inner.pair.x[1].as_py() == 3
+    assert sorted1.inner.pair.x[2].as_py() == 1
+    assert sorted1.inner.pair.y[0].as_py() == 1
+    assert sorted1.inner.pair.y[1].as_py() == 2
+    assert sorted1.inner.pair.y[2].as_py() == 5
+
+    sorted2 = dn.sort_by([("inner.pair.x", "descending")])
+    assert sorted2.inner.id[0].as_py() == "3"
+    assert sorted2.inner.id[1].as_py() == "2"
+    assert sorted2.inner.id[2].as_py() == "1"
+    assert sorted2.inner.pair.x[0].as_py() == 3
+    assert sorted2.inner.pair.x[1].as_py() == 2
+    assert sorted2.inner.pair.x[2].as_py() == 1
+    assert sorted2.inner.pair.y[0].as_py() == 2
+    assert sorted2.inner.pair.y[1].as_py() == 1
+    assert sorted2.inner.pair.y[2].as_py() == 5
+
+
```

Comment:
Please add tests of:
 - Sorting by multiple columns
 - Sorting by ascending explicitly
 - Sorting by ascending in one column, descending in another
 - Using a string other than "ascending" or "descending"
 - Passing a list of column names instead of a list of tuples
 - Attempting to sort by a column name that is not present
 - Sorting an empty table